build-tests/heft-node-everything-esm-module-test/config/copy-static-assets.json

```diff
@@ -1,26 +0,0 @@
-/**
```

Comment:
Was this file just never cleaned up after a previous refactor?

Reply:
Yep! Seems to have been the case.